Source/WebCore/loader/FrameLoader.h

```diff
@@ -457,7 +457,7 @@ class FrameLoader final : public CanMakeCheckedPtr<FrameLoader> {
     void updateNavigationAPIEntries();
     void updateRequestAndAddExtraFields(Frame&, ResourceRequest&, IsMainResource, FrameLoadType, ShouldUpdateAppInitiatedValue, IsServiceWorkerNavigationLoad, WillOpenInNewWindow, Document*);
 
-    bool dispatchNavigateEvent(const URL& newURL, const NavigationAction&, NavigationHistoryBehavior, bool isSameDocument);
+    bool dispatchNavigateEvent(const URL& newURL, const NavigationAction&, NavigationHistoryBehavior, bool isSameDocument, FormState*);
```

Comment:
I hope it is okay to have a default param value of nullptr for the new param, it would make the diff smaller.